Objective-C/Tests/DocumentTest.m

```diff
@@ -2262,6 +2262,62 @@ - (void) testDocumentResaveInAnotherCollection {
     }];
 }
 
+- (void) withOperation: (BOOL (^)(CBLCollection*, CBLMutableDocument*, NSError*)) operation {
```

Comment:
Method name is too generic and needs description. Maybe, `useDifferentCollectionInstance`.